src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -0,0 +1,273 @@
+        }
+        if (statisticAmount >= NUM_ACHIEVEMENT_FOURTH)
+        {
+            TriggerColonyAchievements.triggerFourthAchievement(stat, colony);
+        }
+        if (statisticAmount >= NUM_ACHIEVEMENT_FIFTH)
+        {
+            TriggerColonyAchievements.triggerFifthAchievement(stat, colony);
+        }
+    }
+
+    @Override
+    public List<Advancement> getAchievements()
+    {
+        return new ArrayList<>(colonyAchievements);
+    }
+
+    /**
+     * Get the amount of statistic.
+     *
+     * @param statistic the statistic.
+     * @return amount of statistic.
+     */
+    private int getStatisticAmount(@NotNull final String statistic)
+    {
+        switch (statistic)
+        {
+            case TAG_GUARD_MOBS:
+                return killedMobs;
+            case TAG_MINER_ORES:
+                return minedOres;
+            case TAG_MINER_DIAMONDS:
+                return minedDiamonds;
+            case TAG_BUILDER_HUTS:
+                return builtHuts;
+            case TAG_FISHERMAN_FISH:
+                return caughtFish;
+            case TAG_FARMER_WHEAT:
+                return harvestedWheat;
+            case TAG_FARMER_POTATOES:
+                return harvestedPotatoes;
+            case TAG_FARMER_CARROTS:
+                return harvestedCarrots;
+            case TAG_LUMBERJACK_SAPLINGS:
+                return plantedSaplings;
+            case TAG_LUMBERJACK_TREES:
+                return felledTrees;
+            default:
+                return 0;
+        }
+    }
+
+    /**
+     * increment statistic amount.
+     *
+     * @param statistic the statistic.
+     */
+    private void incrementStatisticAmount(@NotNull final String statistic)
+    {
+        switch (statistic)
+        {
+            case TAG_GUARD_MOBS:
+                killedMobs++;
+                break;
+            case TAG_MINER_ORES:
+                minedOres++;
+                break;
+            case TAG_MINER_DIAMONDS:
+                minedDiamonds++;
+                break;
+            case TAG_BUILDER_HUTS:
+                builtHuts++;
+                break;
+            case TAG_FISHERMAN_FISH:
+                caughtFish++;
+                break;
+            case TAG_FARMER_WHEAT:
+                harvestedWheat++;
+                break;
+            case TAG_FARMER_POTATOES:
+                harvestedPotatoes++;
+                break;
+            case TAG_FARMER_CARROTS:
+                harvestedCarrots++;
+                break;
+            case TAG_LUMBERJACK_SAPLINGS:
+                plantedSaplings++;
+                break;
+            case TAG_LUMBERJACK_TREES:
+                felledTrees++;
+                break;
+            default:
+                break;
+        }
+    }
+
+    @Override
+    public void checkAchievements()
+    {
+        // the colonies size
+        //final int size = this.citizenManager.getCitizens().size();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)